boards/thingy52/include/board.h

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (C) 2017 Feie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    boards_thingy52 Nordic Thingy:52
+ * @ingroup     boards
+ * @brief       Board specific configuration for the Nordic Thingy:52 board
+ * @{
+ *
+ * @file
+ * @brief       Board specific configuration for the Nordic Thingy:52 board
+ *
+ * @author      Hauke Petersen <hauke.petersen@fu-berlin.de>
+ */
+
+#ifndef BOARD_H
+#define BOARD_H
+
+#include "cpu.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
```

Comment:
How about adding an LED configuration? As written [here](https://www.nordicsemi.com/eng/Products/Nordic-Thingy-52) there is a RGB LED on board.

Reply:
There are 6, but all of them are connected through some external GPIO expander chip, for which we (1) don't have a driver (yet), and (2) I am not sure how easily it will be to map the calls to the IO expander to a one-line macro. So this might be added at some point, but is not possible right now.

Reply:
Didn't know that, okay.